pom.xml

```diff
@@ -1,95 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.eclipse.jetty.toolchain</groupId>
-    <artifactId>jetty-toolchain</artifactId>
-    <version>1.6</version>
-    <relativePath>../jetty-toolchain</relativePath>
+    <groupId>org.eclipse.jetty</groupId>
+    <artifactId>jetty-parent</artifactId>
+    <version>26</version>
   </parent>
-  <modelVersion>4.0.0</modelVersion>
   <groupId>org.eclipse.jetty.toolchain</groupId>
   <artifactId>jetty-schemas</artifactId>
   <version>5.3-SNAPSHOT</version>
-  <name>Jetty :: Schemas</name>
   <packaging>jar</packaging>
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-  </properties>
+  <name>Jetty :: Schemas</name>
   <scm>
-    <connection>scm:git:https://github.com/eclipse/jetty.toolchain.git</connection>
-    <developerConnection>scm:git:git@github.com:eclipse/jetty.toolchain.git</developerConnection>
-    <url>https://github.com/eclipse/jetty.toolchain/tree/master/jetty-schemas</url>
+    <connection>scm:git:https://github.com/jetty/${jetty.git.repo}.git</connection>
+    <developerConnection>scm:git:git@github.com:jetty/${jetty.git.repo}.git</developerConnection>
     <tag>HEAD</tag>
+    <url>https://github.com/jetty/${jetty.git.repo}</url>
   </scm>
+  <properties>
+    <jdk.version.minimum>8</jdk.version.minimum>
+    <jetty.git.repo>jetty-schemas</jetty.git.repo>
+  </properties>
   <build>
     <resources>
       <!-- schemas must be on both path to be able to use for 9.4/10 and 11 -->
       <resource>
         <directory>src/main/resources</directory>
       </resource>
       <resource>
-        <directory>src/main/resources/jakarta/servlet</directory>
         <targetPath>javax/servlet</targetPath>
+        <directory>src/main/resources/jakarta/servlet</directory>
       </resource>
     </resources>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>5.1.9</version>
+        <extensions>true</extensions>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <instructions>
+            <Bundle-SymbolicName>org.eclipse.jetty.schemas;singleton:=true</Bundle-SymbolicName>
+            <Bundle-Name>Jetty Servlet Schemas</Bundle-Name>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.7</Bundle-RequiredExecutionEnvironment>
+            <_nouses>true</_nouses>
+            <Export-Package>javax.servlet.resources;version="4.0",
+              javax.servlet.jsp.resources;version="4.0",
+              jakarta.servlet.resources;version="5.0",
```

Comment:
Isn't Servlet 5.0 JDK 11 ?

Reply:
yes. but this pom change is conservative with the current setup.
we can definitely mix with upgrade as well